### PR TITLE
Added dynamic year to copyright.

### PR DESCRIPTION
**Pull request by danielmoll**
Also updated devpack and linters and dangerouslySetInnerHTML on quote so doc can build.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -69,15 +69,21 @@
 .ec-footer .list__item {
   display: inline-block;
   width: 50%;
+  vertical-align: top;
 }
 /* Tablet means 4 columns pretty much everywhere */
 @media screen and (min-width: 650px) and (max-width: 1000px) {
   .ec-footer .list__item { width: 25%; }
```
Comment (umbobabo) [inline index.css]: I prefer mantains the same indentation so I think you should break the line after opening bracket, etc.

File: index.css [hunk 2]
```diff
@@ -69,15 +69,21 @@
...
+    padding-right: 0.5rem;
+  }
+  .ec-footer__list--subs .list__item:nth-child(4n+4) {
+    padding-right: 0;
+  }
 }
 
 
 /* This makes sure that they stretch for flex to pick them up*/
-.ec-footer__list--subs .list__item,
```
Comment (umbobabo) [inline index.css]: Why remove this?

Comment (danielmoll) [inline index.css]: The long link name 'Buy the world in 2016' link had to break as it was overlapping into the next link at 320px. Hence I removed that line.

**Comment by danielmoll** [comment]
@umbobabo updated to display links vertically :)

**Pull request merged**